src/components/GameStatus/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import './styles.scss'
+import usePlayers from '../../hooks/usePlayers'
+
+const GameStatus = ({ currentSquaresState, winnerLine, xIsNext }) => {
+  const { players } = usePlayers()
+  const [firstPlayer, secondPlayer] = players
+
+  const getWinnerText = () => {
+    if (winnerLine !== undefined) {
+      if (currentSquaresState[winnerLine[0]]) {
+        return firstPlayer.playerName
+      } else {
+        return secondPlayer.playerName
+      }
+    } else return null
+  }
```

Comment:
I suggest writing this as guards in functional programming style:

```suggestion
  const getWinnerText = () => {
    if (currentSquaresState[winnerLine[0]]) return firstPlayer.playerName
    
    return secondPlayer.playerName
  }
  
  const winner = winnerLine && getWinnerText()
```

Also, what `currentSquaresState[winnerLine[0]]` stands for? Could you wrap this in a function that makes clear what it returns?